app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -47,6 +47,7 @@ public Media createFromParcel(Parcel parcel) {
     private String thumbUrl;
     protected String imageUrl;
     protected String filename;
+    private String caption;
```

Comment:
I know this file is severally lacking Javadoc, but it is not a reason to not put some. Please explain what the caption is, and especially how it is different from the previous and next properties. Don't worry if it takes several lines. Use the conventional Javadoc comment style.

Reply:
Done